src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -139,9 +139,15 @@ public async Task<ArrayList> WaitForRadioSilenceAsync(TimeSpan? max = null, uint
         /// <returns>The message if one was received; <c>null</c> otherwise</returns>
         public object ReceiveOne(TimeSpan? max = null)
         {
-            MessageEnvelope envelope;
-            if (TryReceiveOne(out envelope, max, CancellationToken.None))
-                return envelope.Message;
+            return ReceiveOneAsync(max).GetAwaiter().GetResult();
```

Comment:
You should not do this, the spec only calls for changes on the `InternalTryReceiveOneAsync` and `InternalTryReceiveOne` methods. It was intentionally done that way to isolate changes in incremental way so we can catch problems as new things are introduced into the new system.